molo/core/content_import/helper.py

```diff
@@ -135,15 +140,29 @@ def import_categories_for_child_language(
             print "no source found for: ", category.source, (
                 SectionPage.objects.all().values('uuid'))
 
+    @classmethod
+    def get_main_language(cls, locales):
+        mains = [locale for locale in locales if locale.get('is_main')]
+
+        if not mains:
+            raise ImportError("No main languages have been given")
+        elif len(mains) > 1:
+            raise ImportError("Cannot have multiple main languages")
+        else:
+            return mains[0]
```

Comment:
This seems like it is solving a separate issue than the one this PR is meant to be solving (handling invalid locales). Maybe we should update the PR title (to help us remember what this PR was for when we look at it some time in the future)?